frontend/packages/ux-editor/src/components/Elements/LayoutSetsContainer.test.tsx

```diff
@@ -57,21 +57,30 @@ describe('LayoutSetsContainer', () => {
     expect(appContextMock.onLayoutSetNameChange).toHaveBeenCalledWith('test-layout-set-2');
   });
 
-  it('should not render add and delete subform buttons when feature is disabled', () => {
-    render();
-    const createSubformButton = screen.queryByRole('button', {
-      name: textMock('ux_editor.create.subform'),
+  it('should render the delete subform button when feature is enabled and selected layoutset is a subform', () => {
```

Comment:
Nice! 🚀 I think it could be a good idea to add a test, or implement in this test, where it checks that the delete subform button is not displayed when a different layout set is selected. What do you think? :)

Reply:
I added new one 🙂